crates/db_schema/Cargo.toml

```diff
@@ -76,6 +75,7 @@ tokio-postgres = { workspace = true, optional = true }
 tokio-postgres-rustls = { workspace = true, optional = true }
 rustls = { workspace = true, optional = true }
 uuid = { workspace = true, features = ["v4"] }
+typed-builder = "0.15.2"
```

Comment:
I thought we wanted to get rid of all these typed builders, and just use Default now.

Reply:
Its still used for many db insert forms. Has the advantage that we can set `#[builder(!default)]` to avoid accidentally creating a community with empty title if that field isnt set.